### PR TITLE
Add ingestion for sources, on both backends

### PR DESCRIPTION
**Pull request by mihaimaruseac**

### CODE REVIEW

**Review by lumjjb — APPROVED** [review]

File: pkg/assembler/backends/neo4j/src.go [hunk 1]
```diff
@@ -442,3 +442,85 @@ func (c *neo4jClient) sourcesNamespace(ctx context.Context, sourceSpec *model.So
...
+				tagStr := record.Values[4].(string)
+				tag = &tagStr
+			}
+			commit := (*string)(nil)
+			if record.Values[3] != nil {
+				// make sure to take a copy
+				commitStr := record.Values[3].(string)
+				commit = &commitStr
+			}
+			nameStr := record.Values[2].(string)
```
Comment (pxp928) [inline pkg/assembler/backends/neo4j/src.go:500]: should we check if `record.Values[2] != nil`? I understand it should never but worth checking.

File: pkg/assembler/backends/neo4j/src.go [hunk 2]
```diff
@@ -442,3 +442,85 @@ func (c *neo4jClient) sourcesNamespace(ctx context.Context, sourceSpec *model.So
...
+			name := &model.SourceName{
+				Name:   nameStr,
+				Tag:    tag,
+				Commit: commit,
+			}
+
+			namespaceStr := record.Values[1].(string)
```
Comment (pxp928) [inline pkg/assembler/backends/neo4j/src.go:507]: same as above

File: pkg/assembler/backends/neo4j/src.go [hunk 3]
```diff
@@ -442,3 +442,85 @@ func (c *neo4jClient) sourcesNamespace(ctx context.Context, sourceSpec *model.So
...
+			namespace := &model.SourceNamespace{
+				Namespace: namespaceStr,
+				Names:     []*model.SourceName{name},
+			}
+
+			srcType := record.Values[0].(string)
```
Comment (pxp928) [inline pkg/assembler/backends/neo4j/src.go:513]: same as above

**Review by pxp928 — APPROVED** [review]
LGTM quick comment

**Pull request merged**